api/clients/v2/verification/cert_verifier.go

```diff
@@ -36,8 +37,12 @@ type CertVerifier struct {
 	logger logging.Logger
 	// go binding around the EigenDACertVerifier ethereum contract
 	certVerifierCaller *verifierBindings.ContractEigenDACertVerifierCaller
-	ethClient          geth.EthClient
+	ethClient          *geth.EthClient
 	pollInterval       time.Duration
+	// storage shared between goroutines, containing the most recent block number observed by calling ethClient.BlockNumber()
+	latestBlockNumber *atomic.Uint64
+	// atomic bool, so that only a single goroutine is polling the internal client with BlockNumber() calls at any given time
+	pollingActive *atomic.Bool
```

Comment:
Nit, if you make these non-pointers, then the zero value is actually a sane instantiated object, I think.

Reply:
Done